README.md

```diff
@@ -2,4 +2,14 @@
 
 The prom-label-proxy enforces a given label in a given PromQL proxy.
 
-This proxy does not perform authentication or authorization, this has to happen before the request reaches this proxy.
+This proxy does not perform authentication or authorization, this has to happen
+before the request reaches this proxy.
+
+Risks outside the scope of this project:
+
+- Tenants controlling ingestion options, e.g: If the system at hand
+  differentiates between two tennants (A and B), and tenant A has the ingestion
+  configuration
+  [honor_labels](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#scrape_config)
```

Comment:
This is true, but for the sake of completeness I would also mention, that if a user controls the configuration (such as relabelling) for a target the same risk exists.

Reply:
Adjusted beginning of list item and added relabelling notice to the end of the item. What do you think @brancz?

Reply:
Sorry for being nit-picky, I think users don't grasp at first that those are distinct things:

* Controlling the scrape config allows people to set labels via relabelling therefore allowing them to pollute other tenants.
* If `honor_labels` is set, that allows targets to pollute the other tenants as Prometheus will respect any labels the target exposes.

Reply:
No worries at all. I am very happy for the feedback!